sdk/storage/Azure.Storage.Common/src/Shared/UserDelegationKeyProperties.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Text;
+
+namespace Azure.Storage.Sas
+{
+    /// <summary>
+    /// Encapsulates the shared properties used by both
+    /// BlobSasQueryParameters and DataLakeSasQueryParameters.
+    /// </summary>
+    internal class UserDelegationKeyProperties
+    {
+        // skoid
+        internal string _objectId { get; set; }
```

Comment:
Properties need to be `PascalCased`.  Drop the underscores.

Reply:
Fixed